catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/TestOpenSearchSource.java

```diff
@@ -13,10 +13,9 @@
  */
 package ddf.catalog.source.opensearch;
```

Comment:
(By which I mean the code looks fine but having tests in place to catch future regressions is always good)